pkgs/top-level/zig-packages.nix

```diff
@@ -0,0 +1,10 @@
+{ lib, pkgs, stdenv, newScope, zig, fetchFromGitHub, buildPackages }:
+
+lib.makeScope newScope (self:
```

Comment:
not sure if this is the best approach, are we planning to package zig libraries? how would that work exactly?

Reply:
I was planning on packing them as Zig doesn't have a package manager right now.

Reply:
How would packaging zig libraries work, would you be able incrementally compile them?

Also, IIRC zig will have a package mananeger in 0.11

Reply:
If zig will have a package manger then I don't see the point in adding packages to nixpkgs, so I will remove zigpackages